docs/Internationalization.md

```diff
@@ -0,0 +1,23 @@
+# Internationalization in Ignite Apps
+
+Ignite currently setup to have Internationalization setup in English, Arabic, and Korean! This is detected on app load and will set your app to that language.
+
+## RTL
```

Comment:
```suggestion
## Right to Left languages (RTL)
```

Wherever we use acronyms and initialisms in the docs, the first usage in the doc should spell it out.